### ISSUE TITLE
Duplicate posts are listed

### ISSUE THREAD

**Issue by mbmasuda** [vwochnik/jekyll-language-plugin#12]
In `jekyll-language-plugin/example/index.html` there is the following code:

```
<ul>
{% for post in site.posts %}
  {% if post.language == page.language %}
    <li><a href="{{ site.baseurl }}{{ post.url }}">{{ post.title }}</a></li>
  {% endif %}
{% endfor %}
</ul>
```

When rendered on my system, the `Welcome to Jekyll!` post is displayed twice on both the English and German pages.  On the German page, the `Willkommen bei Jekyll!` post is displayed once.

I am using Jekyll 3.0.0 on Mac OS X El Capitan 10.11.2 (15C50).

Thank you.

**Comment by vwochnik** [comment]
It is intended that way. The `Welcome to Jekyll` post is listed in both English and German.

**Comment by mbmasuda** [comment]
Thanks for the fast response.  I saw that there is an English version and a German version.  But on the English page, the link goes to the `/en/` version twice, and on the German page, the link goes to the `/de/` version twice.

When I print out the content of `site.posts`, I see that each post is listed twice:

```
<ul>
{% for post in site.posts %}
  <li>{{ post.title }} {{ post.url }}</li>
{% endfor %}
</ul>
```

produces

```
Willkommen bei Jekyll! /de/2015/06/22/willkommen-bei-jekyll.html
Welcome to Jekyll! /en/2015/06/22/welcome-to-jekyll.html
Welcome to Jekyll! /de/2015/06/22/welcome-to-jekyll.html
Welcome to Jekyll! /de/2015/06/22/welcome-to-jekyll.html
Welcome to Jekyll! /en/2015/06/22/welcome-to-jekyll.html
```

I am brand new to Jekyll, so maybe I am misunderstanding how `site.posts` is supposed to work.

**Comment by vwochnik** [comment]
This is intended as the page layout is rendered once in English and once in German regardless if the content is in the same language.

The plugin is built in such a way that pages or posts listed for multiple languages are rendered for each language seperately.

**Comment by vwochnik** [comment]
I am very sorry. I have been able to reproduce the bug and you are indeed correct. While the English post is listed twice, there is only one file for the German translation. I have already made a fix 02baa82.
